pkg/kv/kvserver/kvflowcontrol/replica_rac2/raft_node.go

```diff
@@ -13,6 +13,28 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/roachpb"
 )
 
+func MakeRaftNodeBasicStateLocked(
```

Comment:
Can you add a func comment? It should note that `Replica.mu` must be locked at least for reads.